src/CozyUtils.js

```diff
@@ -144,6 +145,26 @@ class CozyUtils {
       return accountsCollection.destroy(doc)
     }
   }
+
+  async deleteTrigger(accountId) {
+    const triggersCollection = this.client.collection(DOCTYPE_TRIGGERS)
+    const response = await triggersCollection.find({
+      worker: 'konnector'
```

Comment:
There is no route to do a multi field query on the triggers, we can only filter by `worker`.

Reply:
:-°